console/v1/types_console_plugin.go

```diff
@@ -48,6 +48,84 @@ type ConsolePluginSpec struct {
 	// i18n is the configuration of plugin's localization resources.
 	// +optional
 	I18n ConsolePluginI18n `json:"i18n"`
+	// contentSecurityPolicy is a list of Content Security Policy directives for the plugin.
```

Comment:
I'd make a comment here that plugins shouldn't normally need to change this unless they are loading assets from somewhere else. I'd also make a comment about how a plugin can test to see if they have any violations.